doc/source/myrocks/variables.rst

```diff
@@ -2442,6 +2496,16 @@ are positive integers. The block accesses are saved to
 the ``rocksdb_datadir/block_cache_traces/trace_file_name``.
 The default value is ``  ``, an empty string.
 
+.. variable:: rocksdb_trace_queries
+
+   :cli: ``--rocksdb-trace-queries``
+   :dyn: Yes
+   :scope: Global
+   :vartype: String
+   :default: ""
+
+This variable is a trace option string. The format is sampling_frequency:max_trace_file_size:trace_file_name. The sampling_frequency and max_trace_file_size are positive integers. The queries are saved to the rocksdb_datadir/queries_traces/trace_file_name.
+
```

Comment:
Can we check and see if this is further documented in Facebook github wiki, seems useful to maybe add referral if there is more info somewhere.

Reply:
[MyRocks Server Variables](https://github.com/facebook/mysql-5.6/wiki/New-MySQL-RocksDB-Server-Variables)
This page has not been updated since February 2019.

Reply:
:-\